blog/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class User(AbstractUser):
+
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
```

Comment:
you can rely on Django's default behavior for pluralization, and you don't need to use verbose_name_plural in this case